streams/src/test/java/org/apache/kafka/streams/state/internals/StreamThreadStateStoreProviderTest.java

```diff
@@ -76,8 +77,9 @@ public void before() throws IOException {
         final String applicationId = "applicationId";
         properties.put(StreamsConfig.APPLICATION_ID_CONFIG, applicationId);
         properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+        final String stateConfigDir = TestUtils.tempDirectory().getPath();
```

Comment:
@guozhangwang Strictly speaking it doesn't really matter as the directory is deleteOnExit and there is a shutdown hook. However it is good practice to cleanup, so i've done it. Thanks